CONTRIBUTING.md

```diff
@@ -32,6 +32,14 @@ To run tests:
 - :test_tube: Run `yarn check-types` to check TypeScript types.
 - :test_tube: Run `yarn lint` to run the linter (ESLint).
 
+Note that to run `test:e2e`, first ensure that the environment is correctly setup by running the `run_e2e_env.sh` script.
+
+```sh
+GETH_DOCKER_IMAGE=ethereum/client-go:v1.11.6  NETHERMIND_DOCKER_IMAGE=nethermind/nethermind:1.18.0 ./scripts/run_e2e_env.sh start
+```
+
+Similarly, run `yarn download-spec-tests` in both folders `packages/beacon-node` and `packages/validator` before running `yarn test:spec`.
```

Comment:
This sounds annoying, haven't checked but can't we make this a single command and make the `download-spec-tests` smart enough to figure out if it has to download something?

Reply:
Done

Reply:
Top-level command is definitely an improvements, but what I meant specifically was executing it part of `yarn test:spec` but it would not always download the specs but would be smart about it, ie. check if spec tests exists locally and have requested version if not download them, otherwise skip. This is similar to how we do it when downloading the beacon api spec.

(we don't need to do add it in this PR, just a thought how this could be made more dev friendly)

Reply:
https://github.com/ChainSafe/lodestar/issues/6348